src/main/java/com/sweng894/GetVaccinated/api/controller/VaccineController.java

```diff
@@ -0,0 +1,23 @@
+package com.sweng894.GetVaccinated.api.controller;
+
+
+import com.sweng894.GetVaccinated.api.entity.Vaccine;
+import com.sweng894.GetVaccinated.api.repository.VaccineRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class VaccineController {
```

Comment:
What is your plan for rendering content on the page? REST-like endpoints wouldn't be needed for this story if server-side rendering is used. Is the intent to client-side render the content?